README.md

```diff
@@ -1,5 +1,10 @@
 ![Snyk logo](https://snyk.io/style/asset/logo/snyk-print.svg)
 
+    
+[![Known Vulnerabilities](https://snyk.io//test/github/snyk/snyk-mvn-plugin/badge.svg?targetFile=package.json)](https://snyk.io//test/github/snyk/snyk-mvn-plugin?targetFile=package.json)
```

Comment:
Shouldn't it be one `/` after `snyk.io`?

Reply:
Absolutely! Updated